src/Sentry/Platforms/iOS/Extensions/BreadcrumbExtensions.cs

```diff
@@ -1,43 +1,26 @@
+using Sentry.Extensibility;
+
 namespace Sentry.iOS.Extensions;
 
 internal static class BreadcrumbExtensions
 {
-    public static Breadcrumb ToBreadcrumb(this SentryCocoa.SentryBreadcrumb breadcrumb)
-    {
-        var items = breadcrumb.Data as IEnumerable<KeyValuePair<NSString, NSObject?>>;
-        var data = items?.ToDictionary(
-            x => (string)x.Key,
-            x => x.Value?.ToString() ?? "");
-
-        return new Breadcrumb(
+    public static Breadcrumb ToBreadcrumb(this SentryCocoa.SentryBreadcrumb breadcrumb, IDiagnosticLogger? logger) =>
+        new(
             breadcrumb.Timestamp?.ToDateTimeOffset(),
             breadcrumb.Message,
```

Comment:
not that u changed this. but i get null warnings for message and type 
![image](https://user-images.githubusercontent.com/122666/182709477-c547b7c9-650f-4809-8aa2-df131653a27a.png)

should we default them to something?

Reply:
They are nullable.  Intellisense failure.